apps/studio/components/interfaces/QueryPerformanceV2/QueryPerformance.tsx

```diff
@@ -0,0 +1,210 @@
+    {
+      id: QUERY_PERFORMANCE_REPORT_TYPES.MOST_TIME_CONSUMING,
+      label: 'Most time consuming',
+      description: 'Lists queries ordered by their cumulative total execution time.',
+      isLoading: isLoadingMTC,
+      max:
+        (mostTimeConsumingQueries ?? []).length > 0
+          ? Math.max(...(mostTimeConsumingQueries ?? []).map((x: any) => x.total_time)).toFixed(2)
+          : undefined,
+    },
+    {
+      id: QUERY_PERFORMANCE_REPORT_TYPES.MOST_FREQUENT,
+      label: 'Most frequent',
+      description: 'Lists queries in order of their execution count',
+      isLoading: isLoadingMFI,
+      max:
+        (mostFrequentlyInvoked ?? []).length > 0
+          ? Math.max(...(mostFrequentlyInvoked ?? []).map((x: any) => x.calls)).toFixed(2)
+          : undefined,
+    },
+    {
+      id: QUERY_PERFORMANCE_REPORT_TYPES.SLOWEST_EXECUTION,
+      label: 'Slowest execution',
+      description: 'Lists queries ordered by their maximum execution time',
+      isLoading: isLoadingMMF,
+      max:
+        (slowestExecutionTime ?? []).length > 0
+          ? Math.max(...(slowestExecutionTime ?? []).map((x: any) => x.max_time)).toFixed(2)
+          : undefined,
+    },
+  ]
+
+  return (
+    <>
+      <Tabs_Shadcn_
+        defaultValue={page}
+        onValueChange={(value) => {
+          setPage(value as QUERY_PERFORMANCE_REPORT_TYPES)
+          const { sort, search, ...rest } = router.query
+          router.push({ ...router, query: { ...rest, preset: value } })
+        }}
+      >
+        <TabsList_Shadcn_ className={cn('flex gap-0 border-0 items-end z-10')}>
+          {QUERY_PERFORMANCE_TABS.map((tab) => (
+            <TabsTrigger_Shadcn_
+              key={tab.id}
+              value={tab.id}
+              className={cn(
+                'group',
+                'relative',
+                'px-6 py-3 border-b-0 flex flex-col items-start !shadow-none border-default border-t',
+                'even:border-x last:border-r even:!border-x-strong last:!border-r-strong',
+                tab.id === page ? '!bg-surface-200' : '!bg-surface-200/[33%]',
+                'hover:!bg-surface-100',
+                'data-[state=active]:!bg-surface-200',
+                'hover:text-foreground-light',
+                'transition'
+              )}
+            >
+              {tab.id === page && (
+                <div className="absolute top-0 left-0 w-full h-[1px] bg-foreground"></div>
+              )}
+
+              <div className="flex items-center gap-x-2">
+                <span className="">{tab.label}</span>
+                <Tooltip_Shadcn_>
+                  <TooltipTrigger_Shadcn_ asChild>
+                    <InformationCircleIcon className="transition text-foreground-muted w-3 h-3 data-[state=delayed-open]:text-foreground-light" />
+                  </TooltipTrigger_Shadcn_>
+                  <TooltipContent_Shadcn_ side="bottom">{tab.description}</TooltipContent_Shadcn_>
```

Comment:
What do you think about making this `"top"`? That way it won't cover the info text underneath:

<img width="641" alt="Screenshot 2024-04-11 at 11 25 30" src="https://github.com/supabase/supabase/assets/10985857/0a6d990a-10be-41a8-946c-466cbddd7657">

```suggestion
                  <TooltipContent_Shadcn_ side="top">{tab.description}</TooltipContent_Shadcn_>
```